README.rst

```diff
@@ -3,23 +3,63 @@ Dask
 
 |Build Status| |Coverage| |Doc Status| |Discourse| |Version Status| |NumFOCUS|
 
-Dask is a flexible parallel computing library for analytics.  See
+Dask is a flexible parallel computing library for analytics. Visit our website_ or see 
 documentation_ for more information.
 
+Dask enables parallel and distributed computing in Python, allowing you to scale the Python tools you already know and love.
+
+Installation
```

Comment:
Slight preference for just having a link to https://docs.dask.org/en/stable/install.html in this section.